packages/grid/x-grid-modules/src/components/styled-wrappers/getStyles.ts

```diff
@@ -0,0 +1,212 @@
+      cursor: 'pointer',
+    },
+
+    '&  .MuiDataGrid-col-cell.center': {
+      justifyContent: 'center',
+    },
+
+    '&  .MuiDataGrid-col-cell.right': {
+      justifyContent: 'flex-end',
+    },
+
+    '& .MuiDataGrid-col-cell .title': {
+      textOverflow: 'ellipsis',
+      overflow: 'hidden',
+      whiteSpace: 'nowrap',
+    },
+    '& .MuiDataGrid-col-cell .column-separator': {
+      position: 'absolute',
+      right: '-12px',
+      zIndex: 100,
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+    },
+    '& .MuiDataGrid-col-cell .column-separator .icon.separator': {
+      color: '#bdc3c7',
+    },
+    '& .MuiDataGrid-col-cell .column-separator:hover .separator.resizable': {
+      cursor: 'col-resize',
+      color: 'inherit',
+    },
+    '&  .MuiDataGrid-col-cell *': {
+      maxHeight: '56px',
+    },
+    '& .MuiDataGrid-col-cell.checkbox-selection-header-cell .checkbox-input': {
+      padding: '12px',
+    },
+
+    '& .MuiDataGrid-col-cell-wrapper.scroll .MuiDataGrid-col-cell:last-child': {
+      borderRight: 'none',
+    },
+
+    '&  .data-container': {
+      position: 'relative',
+      flexGrow: 1,
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    '&  .window': {
+      position: 'absolute',
+      bottom: '0px',
+      left: '0px',
+      right: 0,
+      overflowX: 'auto',
+    },
+    '&  .window .viewport': {
+      position: 'sticky',
+      top: '0px',
+      left: '0px',
+      display: 'flex',
+      flexDirection: 'column',
+      overflow: 'hidden',
+    },
+    '&  .window .MuiDataGrid-row': {
+      display: 'flex',
+      width: 'fit-content',
+      backgroundColor: '#fff',
+
+      '&:hover': {
+        cursor: 'pointer',
+        backgroundColor: theme.palette.action.hover,
+        // Reset on touch devices, it doesn't add specificity
+        '@media (hover: none)': {
+          backgroundColor: 'transparent',
+        },
+      },
+    },
+    '&  .window .MuiDataGrid-row.odd': {
+      backgroundColor: '#fcfcfc',
+    },
+    '&  .window  .MuiDataGrid-row.selected': {
+      backgroundColor: fade(theme.palette.primary.main, theme.palette.action.selectedOpacity),
+    },
+    '&  .window  .MuiDataGrid-cell': {
+      display: 'block',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      whiteSpace: 'nowrap',
+      padding: theme.spacing(0, 2),
+      fontSize: theme.typography.fontSize,
+      borderBottom: `1px solid ${borderColor}`,
+    },
+    '&  .window  .MuiDataGrid-cell.with-renderer': {
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+    },
+    '&  .with-border': {
+      borderRight: `1px solid ${borderColor}`,
+    },
+    '& .window .MuiDataGrid-cell.right': {
+      textAlign: 'right',
+    },
+    '& .window .MuiDataGrid-cell.center': {
+      textAlign: 'center',
+    },
+    '&  .window  .MuiDataGrid-cell.checkbox-selection-cell .checkbox-input': {
```

Comment:
We need to flatten the selector. Maybe?
```suggestion
    '& .MuiDataGrid-checkboxInput': {
```